app/views/welcome/_logged_in.html.erb

```diff
@@ -0,0 +1,50 @@
+<div class="welcome-container">
+    <div class="welcome-pane">
+        <h1>Welcome, <%= current_user.given_name %>!</h1>
+        <div class="project-button">
+            <%if current_user.eligible_sponsor?%>
+            <%= link_to "Create new project", new_project_path(), class: "btn btn-dark btn-sm" %>
+            <%end%>
+        </div>
+    </div>
+    <div class="detail-pane">
+        <div class="main">
+            <div id="dashboard-nav">
+                <button id="dash-projects" class="tab-nav"> Projects </button>
+                <button id="dash-admin" class="tab-nav"> Administration </button>
+            </div>
+            <% if @dash_session == "project"%>
+                <%= render partial: "projects_listing", locals: { projects: @dashboard_projects.sort_by(&:updated_at).reverse,  table_id: "projects-listing" } %>
+            <% elsif @dash_session == "activity" %>
+                <%= render partial: "recent_activity" %>
+            <% elsif @dash_session == "admin" %>
+                <%= render partial: "admin_listing" %>
+            <% end %>
+        </div>
+        <div class="sidebar">
+            <div class="downloads">
+                <div class="header">
+                    <div><i class="bi bi-download"></i> Latest Downloads</div>
+                </div>
+                <div class="details">
+                    <div class="content">
+                        <div class="container1">
+                        <div class="container2">
+                        <div class="container3">
```

Comment:
These 3 nested divs are to mimic the layout indicated on the Figma prototype. They are empty in content, but they CSS uses them to layout things pretty